Eask

```diff
@@ -0,0 +1,20 @@
+;; -*- mode: eask; lexical-binding: t -*-
+
+(package "renpy"
+         "0.3"
+         "Major mode for editing Ren'Py files")
+
+(website-url "https://github.com/Reagankm/renpy-mode")
+(keywords "languages")
+
+(package-file "renpy.el")
+
+(script "test" "echo \"Error: no test specified\" && exit 1")
+
+;;; Avoid anything non-gnu as dependencies
```

Comment:
I think this comment needs to be `;;` rather than `;;;` to avoid being matched as a level 1 outline-mode heading. Or maybe the comment could just be removed because no additional sources have actually been deleted from the file.

I would also punctuate this as a sentence (ending in a `.`).